### PR TITLE
Emojify😉

### PR DESCRIPTION
**Pull request by binguliki**
# Pull Request for DL-Simplified 💡

## Issue Title : Emojify

- **Info about the related issue (Aim of the project)** : The Goal is of this project is to analyse a given Tweet sentence and predict a an appropriate Emoji for it.
- **Name:** Bingumalla Likith 
- **GitHub ID:** [binguliki](https://github.com/binguliki)
- **Email ID:** blikith86@gmail.com
- **Idenitfy yourself**: `GSSoC'24 | Contributor` 

Closes: #673 

### Describe the add-ons or changes you've made 📃
Emojis are the lifeblood of online expression! This project explored predicting the perfect emoji for your tweets using neural networks. We built two models: a basic one (34% accuracy) and a powerful LSTM (93% accuracy) that considers word order. We fine-tuned the models for optimal performance and analyzed their strengths beyond just accuracy. This paves the way for even more accurate emoji prediction, making your tweets truly shine! 

## Type of change ☑️

What sort of change have you made:

- [x] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [x] Code style update (formatting, local variables)
- [x] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] This change requires a documentation update

## Checklist: ☑️

-  [x] My code follows the guidelines of this project.
- [x] I have performed a self-review of my own code.
- [x] I have commented my code, particularly wherever it was hard to understand.
- [x] I have made corresponding changes to the documentation.
- [x] My changes generate no new warnings.
- [x] I have added things that prove my fix is effective or that my feature works.
- [x] Any dependent changes have been merged and published in downstream modules.

 Hey @abhisheks008 👋 , Can you please check the project and assign me `Level-3` tag for my efforts under` GSSoC'24`.

### CODE REVIEW

**Review by abhisheks008 — CHANGES_REQUESTED** [review]
In the project folder name please remove the emoji and rename it like this, `Emojify - An Emoji Prediction Project using DL`.

Remove the `.gitignore` file.

Either add a web app and showcase the working of the project or create two models for this project. As the existing models are not sufficient enough for this project especially the FCNN model.

**Comment by binguliki** [comment]
@abhisheks008 okay bro i'll update it

**Comment by binguliki** [comment]
Hey bro @abhisheks008 👋,

I've invested a lot of time and effort into developing and fine-tuning another model, incorporating an attention mechanism that has boosted accuracy to 92%. Could you please review the changes and assign the appropriate tag?

**Review by abhisheks008 — APPROVED** [review]
Looks good to me. Approved @binguliki
